docs/src/main/sphinx/release/release-440.md

```diff
@@ -0,0 +1,35 @@
+# Release 440 (21 Feb 2024)
+
+## Web UI
+
+* Fix error when using authentication tokens larger than 4 kB. ({issue}`20787`)
+
+## Delta Lake connector
+
+* Add support for concurrent `INSERT` queries. ({issue}`18506`)
+
+## Hive connector
+
+* Add support for bearer token authentication for a Thrift metastore connection. ({issue}`20371`)
+* Add support for commenting on partitioned columns in the Thrift metastore. ({issue}`20264`)
+* Add support for changing a column's type from `varchar` to `float`. ({issue}`20719`)
```

Comment:
maybe 

```suggestion
* Add support for changing column type from `varchar` to `float`. ({issue}`20719`)
```

also in others

Reply:
Not sure I'm a fan of this change, takes it from proper grammar to improper

Reply:
sounds good